examples/hovorod-cpu/README.md

```diff
@@ -0,0 +1,21 @@
+# Horovod CPU-Only Case
+
+This example shows how to run a cpu-only mpijob.
+
+## How to Build Image
+
+Please download Horovod cpu only dockerfile and then build the image as follows:
+
+```bash
+curl -O https://raw.githubusercontent.com/horovod/horovod/master/Dockerfile.cpu
+# change the default tensorflow version from 2.1.0 to 1.14.0,
```

Comment:
Any reason for this?

Reply:
I think the [example](https://github.com/horovod/horovod/blob/master/examples/tensorflow_mnist.py) in horovod is based on TF 1.x, also I have tested about that, TF 2.0 isn't compatible.